cfme/physical/provider/__init__.py

```diff
@@ -157,5 +157,5 @@ class Add(CFMENavigateStep):
 
     def step(self):
         self.prerequisite_view.toolbar.configuration.item_select(
-            'Add a New Physical Provider'
+            'Add a New Physical Infrastructure Provider'
```

Comment:
**QUESTION**
Does it work with both 5.9 and 5.10?

Reply:
yes, it does.
5.9:
![screenshot_20180730_132935](https://user-images.githubusercontent.com/19214410/43410433-3ac2238c-93fd-11e8-9404-6f534898f258.png)
5.10:
![screenshot_20180730_133204](https://user-images.githubusercontent.com/19214410/43410445-42950048-93fd-11e8-8046-55f379055390.png)